README.md

```diff
@@ -4,23 +4,23 @@ A crowd sourced list of the best music to program to! Please contribute and feel
 
 ## List
 
-|Artist|Album/Playlist|Link|
+|Artist|Name|Type|Link|
 |---|---|---|
```

Comment:
this row needs to have 4 columns, and only has 3 right now. That might be the problem.